### PR TITLE
Todo list new features!

### PR DESCRIPTION
**Pull request by gabrielyea**
## Added functionality to to-do list app:
- Can add tasks.
- Sort functionality added.
- Test the sort function by clicking the sort button!
- Made with webpack.

[Check it out!](https://gabrielyea.github.io/to-do-list/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo
 You can see my page live here: 
-[Check it out!](https://gabrielyea.github.io/module1-capstone/)
+[Check it out!](https://gabrielyea.github.io/to-do-list/)
```
Comment (MahmoudBakr23) [inline README.md:19]: - [x] For completing the professional repo, kindly fix the MIT broken link in the README file down below. 🙏

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo
 You can see my page live here: 
-[Check it out!](https://gabrielyea.github.io/module1-capstone/)
```
Comment (MahmoudBakr23) [inline README.md:16]: - [x] **OPTIONAL**
For a more good-looking README, it'd be great if you attach a screenshot of the app. 😄

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @gabrielyea😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Webpack is well-configured 💯 
- Descriptive PR headline and summary 💯 
- Live demo link 💯
- Code separation 💯 
- Files built with webpack 💯 (JS, HTML, and SCSS) 💯
- Great use of SCSS 💯
- Linters are correct 💯 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (gabrielyea) [reply README.md:19]: Fixed!

File: README.md (hunk 2 above)
Comment (gabrielyea) [reply README.md:16]: Fixed also!

File: README.md (hunk 1 above)
Comment (Davidosky007) [reply README.md:19]: nice job 👍

File: README.md (hunk 2 above)
Comment (Davidosky007) [reply README.md:16]: nicely done 👍

**Comment by Davidosky007** [comment]
Hi @gabrielyea, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 
CR GIF

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**